aiida/common/log.py

```diff
@@ -51,74 +49,77 @@ def filter(self, record):
 
 # The default logging dictionary for AiiDA that can be used in conjunction
 # with the config.dictConfig method of python's logging module
-LOGGING = {
-    'version': 1,
-    'disable_existing_loggers': False,
-    'formatters': {
-        'verbose': {
-            'format': '%(levelname)s %(asctime)s %(module)s %(process)d '
-            '%(thread)d %(message)s',
-        },
-        'halfverbose': {
-            'format': '%(asctime)s <%(process)d> %(name)s: [%(levelname)s] %(message)s',
-            'datefmt': '%m/%d/%Y %I:%M:%S %p',
-        },
-    },
-    'filters': {
-        'testing': {
-            '()': NotInTestingFilter
-        }
-    },
-    'handlers': {
-        'console': {
-            'level': 'DEBUG',
-            'class': 'logging.StreamHandler',
-            'formatter': 'halfverbose',
-            'filters': ['testing']
-        },
-    },
-    'loggers': {
-        'aiida': {
-            'handlers': ['console'],
-            'level': lambda: get_config_option('logging.aiida_loglevel'),
-            'propagate': False,
+def get_logging_config():
```

Comment:
Are the changes in this file necessary for implementing the automation of `__all__`?

Reply:
Yeh otherwise there were messy circular dependency errors.
Two key points here are (a) top-level imports should always flow from `aiida.common` to other modules (since its meant to be a common module) and (b) you should always try to avoid mutable global variables